crates/sargon/src/signing/collector/signatures_collector.rs

```diff
@@ -384,7 +416,13 @@ mod tests {
     impl SignaturesCollector<TransactionIntent> {
         /// Used by tests
         pub(crate) fn petitions(self) -> Petitions<TransactionIntent> {
-            self.state.into_inner().petitions.into_inner()
+            self.state
+                .read()
+                .expect("SignaturesCollector lock was poisoned.")
```

Comment:
The string message that goes into an "expect" should be on the negated format. i.e. you dont write what went wrong, you write why you expected something to be `Ok` or `Some`. So e.g. `SignaturesCollector to not have been poisoned`. or `No poison in SignaturesCollector lock`.

Reply:
so you can probably do find replace all: `"lock was poisoned"` -> `"lock to not be poisoned."` or similar.

Reply:
Kept the same logic we have for profile state https://github.com/radixdlt/sargon/blob/60beabeb1c9fb9be9866fe99d1309824c43dd75d/crates/sargon/src/system/sargon_os/profile_state_holder.rs#L112

Reply:
lets not do it **more wrong** :)